femtools/Checkpoint.F90

```diff
@@ -44,6 +44,7 @@ module checkpoint
   use mesh_files
   use detector_data_types
   use diagnostic_variables
+  use particles, only : checkpoint_particles_loop
```

Comment:
Have you tested checkpointing of the particle info? Does it work? If so, please add a test that checks this functionality.